Rego/EXOConfig.rego

```diff
@@ -20,12 +21,6 @@ ReportDetailsArray(Status, Array1, Array2) = Detail if {
 
 AllDomains := {Domain.domain | Domain = input.spf_records[_]}
```

Comment:
I know this probably isn't your code, but it is good practice to use the assignment operator `:=` instead of the unification operator `=` because it can cause unexpected behavior.